docs/userguide/configuring.rst

```diff
@@ -71,7 +88,7 @@ Here are a series of question to help formulate a suitable configuration:
 |                     | * `HighThroughputExecutor` |                        |
 +---------------------+----------------------------+------------------------+
 
-2. How many nodes do you have to execute them ? What task durations give good performance on different executors?
+2. How many nodes will you use to run them? What task durations give good performance on different executors?
```

Comment:
I've tried not to use "task" here, but in this particular place, I'm not sure that "app" would make sense.

Reply:
This sounds better. As it is, we've used app/task pretty interchangeably. Though I tend to refer to an app executing on a worker as a task.